### ISSUE TITLE
utf8 in powershell core

### ISSUE THREAD

**Issue by kort3x** [curl/curl#13354]
### I did this

`curl https://slftool.github.io/data.json`

then

`(curl https://slftool.github.io/data.json)`

### I expected the following

in both cases Umlaute should work and not brake in the second one

### curl/libcurl version

curl 8.7.1 (x86_64-w64-mingw32) libcurl/8.7.1

### operating system

Win11 22H2
PSVersion: 7.4.1

**Comment by bagder** [comment]
Is the only difference that the second version has parentheses around it or did something fall off?

What does that mean?

> in both cases Umlaute should work

Where are these umlauts?

**Comment by kort3x** [comment]
Yes, only difference are the parentheses. Brakes for other things as well (like the pipeline) but parentheses are enough.

Here is a line with Umlaute and how it looks without parentheses 
 `"stadt": ["Zweibrücken (Deutschland)", "Zwiesel (Deutschland)", "Zwickau (Deutschland)", "Zürich (Schweiz)", "Zabol (Iran)", "Zagreb (Kroatien)"]`

here with parentheses
"stadt": ["Zweibr├╝cken (Deutschland)", "Zwiesel (Deutschland)", "Zwickau (Deutschland)", "Z├╝rich (Schweiz)", "Zabol (Iran)", "Zagreb (Kroatien)"],

**Comment by kort3x** [comment]
Works fine powershell v5

**Comment by bagder** [comment]
This smells like a powershell problem (if any). curl downloads and outputs the data as-is, with no translation or conversion.

**Comment by kort3x** [comment]
i opened an issue there to as i dont know what is happing here.

https://github.com/PowerShell/PowerShell/issues/21456

**Comment by Kangie** [comment]
It seems likely that PowerShell is capturing the output in when inside parens and you're seeing that the console does not default to UTF-8. Not a cURL issue.

**Comment by kort3x** [comment]
fyi

[Console]::OutputEncoding = [Console]::InputEncoding = [System.Text.UTF8Encoding]::new()

helps with external binaries like curl.exe in powershell

**Comment by kort3x** [comment]
not a curl issue but windows
